### DISCUSSION TITLE
version 2.1.1 is crashing on me when in edition/demo mode

### DISCUSSION THREAD

**Discussion by AndyIreland** [cedricp/ddt4all#930] [Q&A]
version 2.1.1 keeps crashing in edition/demo mode when I go to select some ecu's (not all) and I cannot for the life of me find whereabouts in the ddt4all folder the log file is kept - when I can I will post it up . 

![image](https://github.com/cedricp/ddt4all/assets/49172217/1f2d83e5-c528-4b05-83a7-6374b164a3e4)

**Comment by Furtif** [comment]
?

**Comment by KarelSvo** [comment]
@AndyIreland 
The crash message must be read in the terminal window.
The terminal window closes immediately?
Then start the terminal manually.
Use cd to go to the folder C:\Program Files\ddtall\Python38-32 or C:\Program Files(X86)\ddtall\Python38-32
Then start with python.exe ../main.py

**Comment by Furtif** [comment]
..... fine https://github.com/cedricp/ddt4all/discussions/933
